slf4j-api/src/main/java/org/slf4j/helpers/NOPLogger.java

```diff
@@ -25,7 +25,8 @@
 package org.slf4j.helpers;
 
 import org.slf4j.Logger;
-import org.slf4j.helpers.MarkerIgnoringBase;
+
+import java.util.function.Supplier;
```

Comment:
If you use `default` method in Logger - you will have not to implement the methods in NOP

Reply:
Yes. 
We must add the default method (for backward compatibility).
We have to add the method into AbstractLogger (for performance).